include/grpcpp/opencensus_impl.h

```diff
@@ -0,0 +1,51 @@
+/*
+ *
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef GRPCPP_OPENCENSUS_IMPL_H
+#define GRPCPP_OPENCENSUS_IMPL_H
+
+#include "opencensus/trace/span.h"
+
+namespace grpc {
+
+class ServerContext;
+}
+namespace grpc_impl {
+// These symbols in this file will not be included in the binary unless
```

Comment:
Should these comments be in the non-`_impl` file?

Reply:
As I understand, aim is to keep .h file as just a placeholder and move everything including the comments to _impl file. @nicolasnoble  to confirm.

Reply:
I feel like the _impl file is an extra indirection for the reader but it's up to gRPC folks to decide. :)

Reply:
Yeah, we'll try to polish this a bit later. Right now this is kind of the best solution to keep doxygen results pretty.